keyboards/projectkb/alice/rules.mk

```
@@ -1,22 +1,2 @@
-# MCU name
-MCU = STM32F072
+DEFAULT_FOLDER = projectkb/alice/rev2
```

IMO this should point at rev1 to maintain existing behaviour for existing users, with the goal of eventually removing this file entirely.

```suggestion
DEFAULT_FOLDER = projectkb/alice/rev1
```